src/LayerPlan.cpp

```diff
@@ -1108,7 +1118,17 @@ void LayerPlan::addLinesByOptimizer(const Polygons& polygons, const GCodePathCon
         {
             continue;
         }
-        addTravel(p0);
+
+        if(vSize2(getLastPlannedPositionOrStartingPosition() - p0) < line_width_2)
+        {
+            // Instead of doing a small travel that is shorter than the line width (which is generally done at pretty high jerk & move) do a
+            // "fake" extrusion move
+            addExtrusionMove(p0, config, space_fill_type, 0, false, 1.0, fan_speed);
+        } else
```

Comment:
Code style: `else` on a new line.
```suggestion
        }
        else
```